github/Requester.py

```diff
@@ -237,7 +236,7 @@ def __requestEncode(self, cnx, verb, url, parameters, requestHeaders, input, enc
         self.__authenticate(url, requestHeaders, parameters)
         requestHeaders["User-Agent"] = self.__userAgent
         if self.__apiPreview:
-            requestHeaders["Accept"] = "application/vnd.github.moondragon+json"
+            requestHeaders["Accept"] = "application/vnd.github.polaris-preview+json"
```

Comment:
I tested against a dummy repo I do testing in and it does work. I assume that the moondragon preview is not a valid prerelease branch anymore and the content that was on that branch has been merged into their "master"